### PR TITLE
Copy tweaks to if-you-decide-no-to-run section

### PR DESCRIPTION
**Pull request by alexsoble**
# What's this?

+ Soften the list of rhetorical questions with a straightforward statement that we hope Chance will get involved

### CODE REVIEW

**Comment by beamalsky** [comment]
Good call on softening this. Can we keep the [Site Text](https://docs.google.com/document/d/1IX2bi34y2y00EOO6SrGxe55iVo-GhdIweq908Y1geuQ/edit) doc updated with text edits? I have 2 suggested tweaks: 

"Let’s say you decide not to run. That's cool too, but we hope you will still get involved. Will you speak with candidates for mayor about CPS, like you did with Governor Rauner? Will you endorse a candidate who will fight for the South and West sides? Will you help Chicago’s youth participate in Chicago elections, and encourage your fans to vote?"

File: index.html [hunk 1]
```diff
@@ -83,16 +83,16 @@ <h2 id="count"></h2>
                 </div>
             </div>
 
-            <p>Let’s say you decide not to run. Will you get involved and help Chicago’s youth get involved in Chicago elections? Will you speak with candidates for mayor about CPS, like you did with Governor Rauner? Will you endorse a candidate who will fight for the South and West sides? Will you encourage your fans to vote?</p>
-            <p>You don’t have to decide right now. 2019 is kind of a long way away. But the people are screaming for you, and Chicago is hungry for change. Think about it...</p>
+            <p>Let’s say you decide not to run. That's cool too. We hope you will get involved even if you're not a candidate. Will you speak with candidates for mayor about CPS, like you did with Governor Rauner? Will you endorse a candidate who will fight for the South and West sides? Will you encourage your fans to vote? Help Chicago’s youth get involved in Chicago elections?</p>
```
Comment (jeancochrane) [inline index.html]: @beamalsky recommends a slight change here:

> Let’s say you decide not to run. That's cool too, but we hope you will still get involved. Will you speak with candidates for mayor about CPS, like you did with Governor Rauner? Will you endorse a candidate who will fight for the South and West sides? Will you help Chicago’s youth participate in Chicago elections, and encourage your fans to vote?

**Comment by beamalsky** [comment]
Ok jk sorry @alexsoble! Could someone who's more comfortable with github than I am confirm that this is all good/merge it?

File: index.html (hunk 1 above)
Comment (kalilsn) [inline index.html]: @beamalsky i think the idea is we'll merge it after these changes are made

**Comment by kalilsn** [comment]
Added these in separately because of the merge conflict making this a pain... sorry!

**Comment by alexsoble** [comment]
OK thanks @kalilsn !

**Pull request closed without merging**